### DISCUSSION TITLE
[Suggestions] Creature separation window, allow double-clicking on the icons of the number of troops to separate ("2", "3", "4" and "5") to validate the action and close the window directly

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2688] [Ideas]
In fheroes2, it would be nice to allow double-clicking on the icons of the number of troops to separate ("2", "3", "4" and "5") to validate the action and close the window directly.

In fheroes2:

![Separation slots fh2 avec boutons 3 4 5 01](https://user-images.githubusercontent.com/43583994/106055462-b24f7700-60ed-11eb-85b7-9d6b6ae775f7.png)

**Comment by Branikolog** [comment]
If I'm not mistaken currently the game doesn't support double clicking. :(
The only solution available is just selecting an item and then goes second separate click. You can feel the behaviour while using "Town Portal" spell.

**Reply by LeHerosInconnu** [reply]
@Branikolog.

> If I'm not mistaken currently the game doesn't support double clicking. :(
> The only solution available is just selecting an item and then goes second separate click. You can feel the behaviour while using "Town Portal" spell.

Two clicks very close in time can be considered as a double click.
Try for example to double click on the portrait of a hero or on the graphic of a castle in the corresponding list in the adventure map screen, or try to double click on a troop in the hero's screen, it works. :)

**Reply by Branikolog** [reply]
I mean, that game doesn't have a code for this click timing yet. Surely, our team is going to implement it some time. But for now we don't have it. In your examples above we can do clicks with a unlimited pause, which is not a double click, but a simple single click over a selected item, which reveals new window.
